ba/js/lesson4/script.js

```diff
@@ -0,0 +1,123 @@
+//Task 1
+
+const greaterThan = (numA, numB) => (numA > numB ? 1 : numA == numB ? 0 : -1);
+
+console.log(greaterThan(96, 6));
+console.log(greaterThan(0, 6));
+console.log(greaterThan(-4, -4));
+
+//Task 2
+
+const factorial = (n) => (n == 0 || n == 1 ? 1 : n * factorial(n - 1));
+
+console.log(factorial(6));
+
+//Task 3
+
+const threeNumbers = (num1, num2, num3) => +(num1.toString() + num2 + num3);
```

Comment:
look accurately to the description of the task, what should it return?

Reply:
I put a "+" before the function body, so the function returns Number